tensorflow_serving/batching/BUILD

```diff
@@ -114,3 +121,28 @@ cc_test(
         "@org_tensorflow//tensorflow/core:test",
     ],
 )
+
+cc_library(
+    name = "batching_util",
+    srcs = ["batching_util.cc"],
+    hdrs = ["batching_util.h"],
+    visibility = ["//visibility:public"],
```

Comment:
Please don't make it public. If you simply remove this line it will use the default internal visibility.